app/models/exam.rb

```diff
@@ -0,0 +1,12 @@
+class Exam < ApplicationRecord
+  belongs_to :user
+
+  has_one_attached :files
+
+  validates :title, presence: true
+  validates :date, presence: true
+  validates :establishment, presence: true
+  validates :description, presence: true
+  validates :profile_id, presence: true
```

Comment:
Pode ser feito tudo no mesmo validates
```suggestion
  validates :title, :date, :establishment, :description, presence: true
```

Já o profile_id é um "belongs_to :profile", certo? Ele já "valida" que precisa estar presente um Profile associado ao Exam